### PR TITLE
[release-2.9] fix: hack to load modulemd-tools for rhel 84

### PR DESCRIPTION
**Pull request by msdolbey**
**What problem does this PR solve?**:
Installing modulemd-tools from Rocky instead of Centos

**Which issue(s) does this PR fix?**:
https://jira.nutanix.com/browse/NCN-103667

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
n/a, fixing bugs

### CODE REVIEW

File: bundles/redhat8.4/bundle.sh.gotmpl [hunk 1]
```diff
@@ -48,18 +48,18 @@ subscription::defer_unregister
...
+repoquery --archlist=x86_64,noarch --resolve --requires --recursive --enablerepo=kubernetes,rhel-8-for-x86_64-baseos-eus-rpms,codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms  --disablerepo=appstream-rocky $(< packages.txt) | grep -v *.i686  >> reqs.txt
 sed -i 1d reqs.txt # we need to get rid of the first line
 #shellcheck disable=SC2046
 yumdownloader --archlist=x86_64,noarch --setopt=skip_missing_names_on_install=False -x \*i686 $(< reqs.txt)
 #shellcheck disable=SC2046
-yumdownloader --setopt=skip_missing_names_on_install=False -x \*i686 --archlist=x86_64,noarch --resolve --disablerepo=*  --enablerepo=kubernetes,rhel-8-for-x86_64-baseos-eus-rpms,codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms --disablerepo=appstream-centos $(< packages.txt)
+yumdownloader --setopt=skip_missing_names_on_install=False -x \*i686 --archlist=x86_64,noarch --resolve --disablerepo=*  --enablerepo=kubernetes,rhel-8-for-x86_64-baseos-eus-rpms,codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms --disablerepo=appstream-rocky $(< packages.txt)
```
Comment (some-things) [inline bundles/redhat8.4/bundle.sh.gotmpl:62]: Shouldn't need the `--disablerepo=appstream-rocky` here if we have `--disablerepo=*`.

**Review by some-things — APPROVED** [review]
I'm happy with this if cluster lifecycle team is happy.

Agnostic as to what the 'proper' approach is here (e.g., rocky repos vs. centos vault vs. RH release set to a different version via RHSM.

File: bundles/redhat8.4/bundle.sh.gotmpl (hunk 1 above)
Comment (dkoshkin) [reply bundles/redhat8.4/bundle.sh.gotmpl:62]: ➕

File: bundles/redhat8.4/bundle.sh.gotmpl [hunk 2]
```diff
@@ -48,18 +48,18 @@ subscription::defer_unregister
 subscription-manager repos --enable rhel-8-for-x86_64-baseos-eus-rpms
 subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
 subscription-manager repos --enable rhel-8-for-x86_64-appstream-rpms
-yum --disablerepo=appstream-centos -y install gettext yum-utils createrepo dnf-utils
+yum --disablerepo=appstream-rocky -y install gettext yum-utils createrepo dnf-utils
```
Comment (dkoshkin) [inline bundles/redhat8.4/bundle.sh.gotmpl:51]: I think we want this?
```suggestion
yum -y install gettext yum-utils createrepo dnf-utils modulemd-tools
```

Comment (dkoshkin) [reply bundles/redhat8.4/bundle.sh.gotmpl:51]: nvm, its being installed further down.

File: bundles/redhat8.4/bundle.sh.gotmpl (hunk 1 above)
Comment (msdolbey) [reply bundles/redhat8.4/bundle.sh.gotmpl]: This is done - thanks for the suggestion!

**Review by dkoshkin — APPROVED** [review]
RHEL 8.4 tests are looking good.

**Pull request merged**